enterprise/server/util/ociconv/ociconv.go

```diff
@@ -39,12 +38,7 @@ var (
 )
 
 func init() {
```

Comment:
nice! I think this could be moved out of an init function now:

```go
var isRoot = os.Getuid() == 0
```

(GH wouldn't let me make a clickable suggestion here for some reason :thinking: )

Reply:
> (GH wouldn't let me make a clickable suggestion here for some reason 🤔 )

Yeah, github doesn't let you comment (and thus suggest) on lines outside the diff context.

> I think this could be moved out of an init function now:

Good point! fixed.